snuba/datasets/configuration/storage_builder.py

```diff
@@ -76,7 +76,7 @@ def __build_readable_storage_kwargs(config: dict[str, Any]) -> dict[str, Any]:
 
 
 def build_stream_loader(loader_config: dict[str, Any]) -> KafkaStreamLoader:
-    processor = CONF_TO_PROCESSOR[loader_config["processor"]]()
+    processor = DatasetMessageProcessor.get_from_name(loader_config["processor"])()
```

Comment:
I didn't add init params to this yet. The ErrorsProcessor will eventually require this though.

Reply:
can we add a ticket to support init args for `DatasetMessageProcessor(s)`? Also, you may want to call `from_kwargs` for completeness here

Reply:
Done and done.